modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -21,6 +21,9 @@
   "À϶￥🌭∮⇒⇔¬β∀₂⌀ıəˈ⍳⍴V)"                           \
   "═€ίζησθლბშიнстемองจึองታሽ።ደለᚢᛞᚦᚹ⠳⠞⠊⠎▉▒▒▓😃"
 
+#define MMD_TEST_DESC_LOCALE "C"
```

Comment:
I don't think this is necessary as "C" refers to the default locale. It's unlikely to change. I personally think the readability is being lost by such a huge macro, but I think we'll let this stay. 

@sgallagher What do you think?

Reply:
So should I just write "C" as an argument rather then create macro?